question5/README.md

```diff
@@ -3,5 +3,8 @@
 (K&R Exercise 5-8) There is no error checking in <code>day_of_year</code> or <code>month_day</code>. Remedy this defect and verify that all of the cases are covered with examples.
 
 Compile Steps:
-
+gcc q5.c
 Output:
+1994 8 31 is 1994 243
+1994 243 is 1994 8 31
```

Comment:
would have been useful for you to show the error cases; output-1
output: 9
quality: 10
19/20